services/graph/pkg/service/v0/driveitems_test.go

```diff
@@ -731,36 +731,37 @@ var _ = Describe("Driveitems", func() {
 			_, ok := res.GetExpirationDateTimeOk()
 			Expect(ok).To(BeFalse())
 		})
-		It("updates the share permissions with changing the role", func() {
-			getPublicShareMockResponse.Share = nil
-			getPublicShareMockResponse.Status = status.NewNotFound(ctx, "not found")
-			updateShareMock := gatewayClient.On("UpdateShare",
-				mock.Anything,
-				mock.MatchedBy(func(req *collaboration.UpdateShareRequest) bool {
-					return req.GetShare().GetId().GetOpaqueId() == "permissionid"
-				}),
-			)
-			updateShareMock.Return(updateShareMockResponse, nil)
-
-			driveItemPermission.SetRoles([]string{unifiedrole.NewViewerUnifiedRole(true).GetId()})
-			body, err := driveItemPermission.MarshalJSON()
-			Expect(err).To(BeNil())
-			svc.UpdatePermission(
-				rr,
-				httptest.NewRequest(http.MethodPatch, "/", strings.NewReader(string(body))).
-					WithContext(ctx),
-			)
-			Expect(rr.Code).To(Equal(http.StatusOK))
-			data, err := io.ReadAll(rr.Body)
-			Expect(err).ToNot(HaveOccurred())
-
-			res := libregraph.Permission{}
-
-			err = json.Unmarshal(data, &res)
-			Expect(err).ToNot(HaveOccurred())
-			_, ok := res.GetRolesOk()
-			Expect(ok).To(BeTrue())
-		})
+		// that is resharing test. Please delete after disable resharing feature
```

Comment:
do we want to merge the PR with that commented code?

Reply:
yes, we want to delete all resharing code and all resharing unit tests after merging this PR
cc @kobergj

Reply:
The commented out code will be removed in following PRs 
see here https://github.com/owncloud/ocis/issues/8744